### PR TITLE
Review: Update German translation for Git v2.37

### PR DESCRIPTION
**Pull request by ruester**
@ralfth and @PhillipSz: Please have a look at my suggestions for the German translation of Git v2.37. Only the commit b3075795cf needs to be reviewed, the commit 80891eb836 is the automatically generated one to update the po file.
There was also a change for the location metadata of the po files, they should now be "location-less" to reduce noise and size of the Git repository, see the po/README.md for more details. The commit for removing the location lines is 86c5b80372.

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -12853,13 +12846,21 @@ msgid ""
 "Use `git rebase --abort` to terminate current rebase.\n"
 "Or downgrade to v2.33, or earlier, to complete the rebase."
 msgstr ""
+"`rebase --preserve-merges` (-p) wird nicht mehr unterstützt.\n"
+"Nutze `git rebase --abort`, um den aktuellen Rebase abzubrechen.\n"
+"Oder gehe auf Version v2.33, oder früher, zurück, um den Rebase "
```
Comment (ralfth) [inline po/de.po]: Maybe we want to replace the last space with \n.

File: po/de.po [hunk 2]
```diff
@@ -18169,37 +18165,35 @@ msgid "no libc information available\n"
...
-"konnte die folgenden künstlichen Vorgänger (\"grafts\") nicht konvertieren:\n"
-"%s"
+msgstr "konnte nicht zum Breitzeichensatz konvertieren: '%s'"
 
 #: compat/fsmonitor/fsm-health-win32.c:162
-#, fuzzy, c-format
+#, c-format
 msgid "BHFI changed '%ls'"
-msgstr "Kann nicht nach '%s' wechseln."
+msgstr "BHFI has sich verändert '%ls'"
```
Comment (ralfth) [inline po/de.po]: I think this should read "BHFI veränderte '%ls'".

**Review by ralfth — CHANGES_REQUESTED** [review]
Very good translation. Thank you!

Just found 2 small things that can be improved.

File: po/de.po (hunk 2 above)
Comment (ruester) [reply po/de.po]: yes, thanks! :see_no_evil:

**Comment by ruester** [comment]
I also added the commit for changing the team leader. @PhillipSz I've discussed this with Ralf and now I also want to inform you that I will retire the team lead for the German translations. But I will stay with you as a reviewer :wink:

**Review by ralfth — APPROVED** [review]
Looks good to me. Thank you!

Reviewed-by: Ralf Thielow <ralf.thielow@gmail.com>

**Comment by ruester** [comment]
https://github.com/git-l10n/git-po/pull/642

**Pull request closed without merging**